ethereum/statetransition/src/main/java/tech/pegasys/artemis/statetransition/util/AttestationUtil.java

```diff
@@ -73,11 +75,11 @@ private static boolean isAttestationCurrentEpoch(
       for (PendingAttestationRecord record : current_epoch_boundary_attestations) {
         if (record
                 .getData()
-                .getEpoch_boundary_hash()
+                .getEpoch_boundary_root()
                 .equals(
                     BeaconStateUtil.get_block_root(
-                        state, record.getData().getSlot() - Constants.EPOCH_LENGTH))
-            && record.getData().getJustified_slot().equals(state.getJustified_epoch()))
+                        state, record.getData().getSlot().intValue() - Constants.EPOCH_LENGTH))
```

Comment:
Same concerns as previous intValue. Subtraction, however, is bitwise identical in conventional and 2s complement arithmetic, so while preferred it's not required to use UnsignedLong's minus function.